### PR TITLE
Global gardener secret config

### PR DESCRIPTION
**Pull request by koala7659**
This PR contains helm chart changes to unify the configuration for gardener kubeconfig secret and configuration used by both Kyma Environement Broker and Provisioner. Following values have been moved to global chart level (file values.yml) and applied in respective deployments.

```
  gardener:
         project: 
         kubeconfigPath:
         kubeconfig:
         secretName: 
         manageSecrets:
```

This PR will also require changes in the global override configuration for example on mps-dev cluster

### CODE REVIEW

File: chart/compass/values.yaml [hunk 1]
```diff
@@ -150,6 +150,13 @@ global:
       name: compass-istio-gateway
       namespace: compass-system
 
+  gardener:
+    project: "kyma-dev" # Gardener project connected to SA
+    kubeconfigPath: "/gardener/kubeconfig/kubeconfig"
```
Comment (piotrmiskiewicz) [inline chart/compass/values.yaml]: this should not be a parameter, let's hardcode it in the chart (it is connected with volume mount)

File: chart/compass/values.yaml [hunk 2]
```diff
@@ -34,15 +34,15 @@ global:
       version: "PR-913"
     provisioner:
       dir:
-      version: "PR-913"
+      version: "PR-953"
```
Comment (Szymongib) [inline chart/compass/values.yaml]: You are not making any changes in the `components` directory, so the images will not be created. The same is true for KEB and Provisioner tests.

File: chart/compass/values.yaml (hunk 1 above)
Comment (koala7659) [reply chart/compass/values.yaml]: I have changed it to be built from configuration variable  .Values.global.gardener.kubeconfigMountPath

File: chart/compass/values.yaml (hunk 2 above)
Comment (koala7659) [reply chart/compass/values.yaml]: True. Only charts are updated

**Comment by crabtree** [comment]
The component was moved to the https://github.com/kyma-project/control-plane repository.

**Pull request closed without merging**